x-pack/plugins/observability_solution/observability_logs_explorer/public/components/logs_explorer_top_nav_menu.tsx

```diff
@@ -137,13 +138,14 @@ const StatefulTopNav = () => {
         chrome.setBreadcrumbsAppendExtension(previousAppendExtension);
       }
     };
-  }, [chrome, i18n, previousAppendExtension, theme]);
+  }, [chrome, i18nStart, previousAppendExtension, theme]);
 
   return (
     <HeaderMenuPortal setHeaderActionMenu={setHeaderActionMenu} theme$={theme.theme$}>
       <EuiHeaderSection data-test-subj="logsExplorerHeaderMenu">
         <EuiHeaderSectionItem>
           <EuiHeaderLinks gutterSize="xs">
+            <CreateSloLinkForValidState observability={observability} />
```

Comment:
we should do the same for serverless. Search for `ServerlessTopNav` in this file and add `<CreateSloLinkForValidState observability={observability} />` there as well.

Reply:
I made the change already and I am now deploying it to serverless https://github.com/elastic/observability-test-environments/issues/21720. I'll test it there, once it is deployed